pkg/bitcoin/htlc/timers.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from "node-fetch"
+
+/**
+ * Set up a Timers class prompting the user for a desired HTLC timer specified in blocks. One block equals roughly 10 minutes.
+ * Default timer is set at 6 blocks (approx. 1 hour).
+ * Minimum timer equals 4 blocks (approx. 0.75 hours).
+ * Maximum timer equals 4000 blocks (approx. 667 hours).
+ */
+
```

Comment:
remove empty line

please please please please use "npm run fmt", it's there for a reason